### ISSUE TITLE
Testing Accuracy mismatch for Kinetics 400 on SLOWFAST_4x16_R50.yaml

### ISSUE THREAD

**Issue by msingh27** [facebookresearch/SlowFast#138]
Hi,

Thanks a lot for making your code public for the paper SlowFast.

I am facing an issue where I am training Kinetics400 dataset using SLOWFAST_4x16_R50.yaml. 
Following is the log message at the end of 196 epoch: 
**Train Accuracy:**
[INFO: logging.py:   67]: json_stats: {"_type": "train_epoch", "epoch": "196/196", "eta": "0:00:00", "loss": 0.978663, "lr": 0.000000, "mem": 7121, "time_diff": 1.054816, "top1_err": 24.498346, "top5_err": 9.274414}_
**Val Accuracy:**
[INFO: logging.py:   67]: json_stats: {"_type": "val_epoch", "epoch": "196/196", "mem": 7121, "min_top1_err": 40.216512, "min_top5_err": 19.470862, "time_diff": 0.818073, "top1_err": 40.555443, "top5_err": 19.470862}_

Somehow the model is getting severely overfit as the mismatch gap between Train and Val accuracy starts to grow after 150 epoch.
**Train Accuracy:**
[INFO: logging.py:   67]: json_stats: {"_type": "train_epoch", "epoch": "150/196", "eta": "2 days, 3:53:52", "loss": 1.701233, "lr": 0.012986, "mem": 7121, "time_diff": 1.107962, "top1_err": 40.810829, "top5_err": 18.658364}_
**Val Accuracy:**
[INFO: logging.py:   67]: json_stats: {"_type": "val_epoch", "epoch": "150/196", "mem": 7121, "min_top1_err": 47.399838, "min_top5_err": 23.568393, "time_diff": 0.848544, "top1_err": 47.399838, "top5_err": 23.568393}_

I trained this model on 8-GPU Tesla V100 system using the given config.

Can you please look into this issue?

Thanks,
Mayank

**Comment by takatosp1** [comment]
Hi,
Thanks for playing with PySlowFast. The accuracy you reported is the one-crop accuracy for validation. Could you please share me what is the number you got for testing stage?

Beside of that, someone has posted the link to kinetics data here #42. You might also want to download the data in you found there are too many video missing from your side.

**Comment by msingh27** [comment]
Hi @takatosp1,

Thanks for pointing out to test for 30-crop accuracy. 

I got 74.54 and 91.27 as the top1_acc and top5_acc now 👍 

_[INFO: logging.py:   67]: json_stats: {"split": "test_final", "top1_acc": "74.54", "top5_acc": "91.27"}_

**In the paper, it is reported to be 75.6 and 92.1 for SLOWFAST_4x16_R50. Does it need some kind of finetuning to further reach up to this accuracy?**

Also, thanks for pointing to the dataset. My training logs from that dataset only :D. 
Following are the details about the number of files in my dataset.

_[INFO: kinetics.py:   73]: Constructing Kinetics train...
[INFO: kinetics.py:  108]: Constructing kinetics dataloader (size: 234619) from ./train.csv
[INFO: kinetics.py:   73]: Constructing Kinetics val...
[INFO: kinetics.py:  108]: Constructing kinetics dataloader (size: 19761) from ./val.csv_

Thanks for the prompt reply,
Mayank

**Comment by takatosp1** [comment]
Hi 
The dataset pointed in #42 is also smaller than what we had, I am expecting to see larger than 0.5% difference.
Could you test my SF pretrain model and share me what is the accuracy with the val set from #42? (people told me there was two val files in from the dropbox, you might try both)

Thanks!

**Comment by msingh27** [comment]
Hi,

I tried testing with your SF 4x16_R50 pretrained model and I got "75.19", "top5_acc": "91.47" on my 
Val set that was taken from #42 which has 19761 videos.  (I only found 2 folders in #42 namely train_256  and val_256).

Can you please share the number the files of your val set and train set?

Thanks  :D
